e2e_tests/footer_tests.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from playwright.sync_api import Page, expect
+
+# Define list of footer links and their expected URLS
+
+footer_links = [
+    (
+        "Terms of use",
+        "/terms-of-use",
+        "Terms of use – AYR - Access Your Records – GOV.UK ",
+    ),
+    (
+        "Privacy",
+        "/privacy",
+        "Privacy notice – AYR - Access Your Records – GOV.UK ",
+    ),
+    (
+        "Cookies policy",
+        "/cookies",
+        "Cookies – AYR - Access Your Records – GOV.UK",
+    ),
+    (
+        "Accessibility",
+        "/accessibility",
+        "Accessibility statement – AYR - Access Your Records – GOV.UK",
+    ),
+]
+
+
+@pytest.fixture
+def setup_page(page: Page):
+    page.goto("/")
+    yield page
+
+
+@pytest.mark.parametrize(
+    "link_text, expected_url, expected_title", footer_links
+)
+def test_footer_links(setup_page, link_text, expected_url, expected_title):
```

Comment:
I this e2e test is fine

but just highlighting the fact that when we think they are sufficient, we can also make lighterweight tests that just test the html response of the flask views like:

```
def test_footer(client):
     assert f"<a class="govuk-footer__link" href="/cookies">Cookies policy</a>" in client.get("/")
```